server/scripts/esrupgrades/esr.5.37-7.8.mysql.up.sql

```diff
@@ -909,7 +912,10 @@ BEGIN
 
 	IF DropParentId THEN
 		SET DropParentIdQuery = 'DROP COLUMN ParentId';
-		UPDATE Posts SET RootId = ParentId WHERE RootId = '' AND RootId != ParentId;
+		SELECT COUNT(*) FROM Posts WHERE RootId != ParentId INTO UpdateRootId;
```

Comment:
Any reason we don't want to add the `RootId = ''` part into the select query?

Reply:
Gentle ping @agarciamontoro :)

Reply:
Yes, it came from the customer. Maybe that was the difference, indeed. We can ask the customer again about it.

Reply:
@agarciamontoro any updates on this one?

Reply:
Nope, and I don't think we'll get any. The customer successfully finished the upgrade (including this change), so they will probably move on to other things.

I still think we should still merge this to reflect what the customer used in production. Thoughts?

Reply:
Hmm, if there is no case the `RootId` gets an empty string, (default value is alreadt `NULL`) it should be good to go.